### ISSUE TITLE
Support for EFS

### ISSUE THREAD

**Issue by aarrowood** [wallix/awless#149]
Are there plans to support creating, listing, etc EFS volumes?

**Comment by simcap** [comment]
Yes we will support it. Basically CRUD on `filesystem`, `mounttarget`, with the usual simplification if we can do any. We might be able to do it either in upcoming 0.1.5 or the next one 0.1.6
